modules/electrophysiology_browser/jsx/electrophysiologySessionView.js

```diff
@@ -0,0 +1,356 @@
+            name: '',
+            task: {
+              frequency: {
+                sampling: '',
+                powerline: '',
+              },
+              channel: [
+                {
+                  name: '',
+                  value: '',
+                },
+                {
+                  name: '',
+                  value: '',
+                },
+                {
+                  name: '',
+                  value: '',
+                },
+                {
+                  name: '',
+                  value: '',
+                },
+              ],
+              reference: '',
+            },
+            details: {
+              task: {
+                description: '',
+              },
+              instructions: '',
+              eeg: {
+                ground: '',
+                placement_scheme: '',
+              },
+              trigger_count: '',
+              record_type: '',
+              cog: {
+                atlas_id: '',
+                poid: '',
+              },
+              institution: {
+                name: '',
+                address: '',
+              },
+              misc: {
+                channel_count: '',
+              },
+              manufacturer: {
+                name: '',
+                model_name: '',
+              },
+              cap: {
+                manufacturer: '',
+                model_name: '',
+              },
+              hardware_filters: '',
+              recording_duration: '',
+              epoch_length: '',
+              device: {
+                version: '',
+                serial_number: '',
+              },
+              subject_artifact_description: '',
+            },
+            downloads: [
+              {
+                type: 'physiological_file',
+                file: '',
+              },
+              {
+                type: 'physiological_electrode_file',
+                file: '',
+              },
+              {
+                type: 'physiological_channel_file',
+                file: '',
+              },
+              {
+                type: 'physiological_task_event_file',
+                file: '',
+              },
+              {
+                type: 'all_files',
+                file: '',
+              },
+              {
+                type: 'physiological_fdt_file',
+                file: '',
+              },
+            ],
+          },
+        },
+      ],
+    };
+
+    // Bind component instance to custom methods
+    this.collectParams = this.collectParams.bind(this);
+    this.fetchData = this.fetchData.bind(this);
+  }
+
+  /**
+   * Fetch data when component mounts.
+   */
+  componentDidMount() {
+    this.collectParams();
+    this.fetchData();
+  }
+
+  /**
+   * Post-Render when we can access the DOM.
+   *
+   * @param {object} prevProps
+   * @param {object} prevState
+   */
+  componentDidUpdate(prevProps, prevState) {
+    console.log('componentDidUpdate');
```

Comment:
I don't think this should be here, and then removing it makes another empty function that should be removed..

Reply:
removed